geowebcache/core/src/main/java/org/geowebcache/storage/blobstore/file/FileBlobStore.java

```diff
@@ -497,7 +512,7 @@ public void put(TileObject stObj) throws StorageException {
         }
     }
 
-    private File getFileHandleTile(TileObject stObj, boolean create) throws StorageException {
+    private File getFileHandleTile(TileObject stObj, Runnable upmFunction) throws StorageException {
```

Comment:
You are using it to update the parameter map, but in the context of this method, it's just something happening as the parent tile folder is created (could be anything depending on the needs of the caller). I'd call it "onTileFolderCreation" instead.